content/docs/2.8/concepts/scaling-deployments.md

```diff
@@ -226,6 +226,37 @@ metadata:
 
 The presensce of this annotation will pause autoscaling no matter what number of replicas is provided. The above annotation will scale your current workload to 0 replicas and pause autoscaling. You can set the value of replicas for an object to be paused at to any arbitary number. To enable autoscaling again, simply remove the annotation from the `ScaledObject`definition.
 
+### `Activating` and `Scaling`
+
+There are use-cases where the activating value (0-1 and 1-0) is totally different than 0, such as workloads scaled with the Prometheus scaler where the values go from -X to X. To give a consisten solution to this problem, KEDA has 2 different phases during the autoscaling process.
+
+- **Activation phase**
+- **Scaling phase**
+
+#### **Activating pahse**
+
+The activating (or deactivating) phase is the moment when KEDA (operator) has to decide if the workload should be scaled from/to zero. KEDA is the responsible for this action based on the result of the scaler `IsActive` function and only applies in from/to 0 scaling.
+
+#### **Scaling pahse**
+
+The scaling phase is the moment when KEDA has decided to scale out to 1 instance and now is the HPA controller who takes the scaling decisions based on the configuration defined in the generated HPA (from ScaledObject data) and the metrics exposed by KEDA (metrics server). This phase applies the scaling from 1-N and N-1.
+
+#### **Thresholds**
+
+KEDA allows you to specify different values for each scenario:
+
+- **Activation:** Defines when the scaler is active or not and scales from/to 0 based on it.
+- **Scaling:** Defines the target value to scale the workload from 1 to _n_ instances and vice versa. To achieve this, KEDA passes the target value to the Horizontal Pod Autoscaler (HPA) and the built-in HPA controller will handle all the autoscaling.
```

Comment:
We should use the configuration values here, for example:
```suggestion
- `activationThreshold`:  Defines when the scaler is active or not and scales from/to 0 based on it.
- `threshold`:  Defines the target value to scale the workload from 1 to _n_ instances and vice versa. To achieve this, KEDA passes the target value to the Horizontal Pod Autoscaler (HPA) and the built-in HPA controller will handle all the autoscaling.
```

Reply:
The problem is that we don't have specific values for that, they depend on the scaler, they can be queueLenght, threshold, value, lag... The constraint is that the activation value name will be the scaling value appending the prefix `activantion`, that's why I preferred to use these names

Reply:
But we can use `threshold` as the word for explaining the concept. WDYT?

Reply:
yeah, let's keep the **Activation** and **Scaling** here

Reply:
No no, don't worry, this will be the base for the change so it's better to discuss it and take the best possible approach.
I prefer to make this clearest as possible before merging it and starting with these changes

Reply:
To clarify this, I have made it bold the paragraph where this is explained. I think that now it's clearer